python/cudnn/gemm/frost/kernel_templates/sm107_block_scale_matmul_1ctamma.py

```
@@ -0,0 +1,1440 @@
+        tile_n = init_tile_n
+        tile_l = init_tile_l
+        is_valid = cutlass.Int32(1)
+        clc_full_phase_epi = cutlass.Int32(0)
+
+        if cutlass.const_expr(mma_inst_shape_mnk[0] == 64):
+            row_id_with_warp_offset = base_row_id
+        else:
+            row_id_with_warp_offset = base_row_id + warp_idx * 32
+
+        # One M block's accumulator columns are contiguous.
+        subtile_cnt = cute.ceil_div(epi_cols_per_mma_m, 32)
```

Ruff (0.16.1)

[error] Undefined name `epi_cols_per_mma_m`

(F821)